hw/top_earlgrey/syn/asic.constraints.sdc

```diff
@@ -0,0 +1,297 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Generic constraints file for simple testsynthesis flow
+
+# Note that we do not fix hold timing in this flow
+set SETUP_CLOCK_UNCERTAINTY 0.5
+set CLK_PERIOD_FACTOR 0.95 ;# clock period over constraining factor
+puts "Applying constraints for top level"
+
+# Note: the netlist does include pads at this level, but not all IO interfaces
+# have been properly constrained yet. The clocks are generated inside AST and
+# for the purpose of test synthesis, these clock nets are just set to ideal networks.
+
+#####################
+# main clock        #
+#####################
+set MAIN_CLK_PIN ast_wrapper/i_ast/i_sys_clk/i_sys_osc/sys_clk_o
```

Comment:
this might be corrected in later commits already..but `ast_wrapper` doesn't exist anymore.

Reply:
yeah that's right, this is again just carried over 1:1 from Bronze. 
several of these SDC constraints need to be reworked.

Reply:
I have aligned all these SDC constraints such that they work with the current state of the design.